saw-central/src/SAWCentral/Bisimulation.hs

```diff
@@ -182,14 +187,18 @@ buildCompositionSideCondition bc innerBt = do
   sc <- getSharedContext
   let outerBt = bcTheorem bc
 
-  lhsOuterState <- io $ scLocalVar sc 0        -- g_lhs_s
-  rhsOuterState <- io $ scLocalVar sc 1        -- g_rhs_s
+  inputEC <- io $ importEC sc "input" (bcInputType bc)
+  lhsOuterStateEC <- io $ importEC sc "lhsState" (bisimTheoremLhsStateType outerBt)
+  rhsOuterStateEC <- io $ importEC sc "rhsState" (bisimTheoremRhsStateType outerBt)
+
+  lhsOuterState <- io $ scExtCns sc lhsOuterStateEC  -- g_lhs_s
+  rhsOuterState <- io $ scExtCns sc rhsOuterStateEC  -- g_rhs_s
```

Comment:
This isn't your fault (it was like this before this PR), but it's a bit odd that the SAWCore code uses the names `lhsState`/`rhsState` for these variables, but the comments refer to them as `g_lhs_s`/`g_rhs_s`. I wonder if the former names should be changed to be in sync with the latter (or vice versa).

(Similar questions apply to every other use of `importEC`.)

Reply:
One of the variables names from the comments was called "in", which we can't use...